src/components/Divider/docs/index.story.tsx

```diff
@@ -12,6 +42,71 @@ export default {
   componentProps: {
     'data-hook': 'storybook-Divider',
   },
+  sections: [
+    header(),
+    tabs([
+      tab({
+        title: 'Usage',
+        sections: [
+          description('A Divider'),
```

Comment:
You could be a little more informative... 😅
For example "Divider is a component helping to separate elements into sections" or something like that

Reply:
😅